staging/src/k8s.io/pod-security-admission/manifests/webhook.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: podsecurity
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: podsecurity-webhook-serviceaccount
+  namespace: podsecurity
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: podsecurity-webhook-clusterrole
+rules:
+  - apiGroups: [""]
+    resources: ["pods"]
+    verbs: ["get", "watch", "list"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: podsecurity-webhook-clusterrolebinding
+subjects:
+  - kind: ServiceAccount
+    name: podsecurity-webhook-serviceaccount
+    namespace: podsecurity
+roleRef:
+  kind: ClusterRole
+  name: podsecurity-webhook-clusterrole
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: podsecurity-webhook-deployment
+  labels:
+    app: podsecurity-webhook
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: podsecurity-webhook
+  template:
+    metadata:
+      labels:
+        app: podsecurity-webhook
+    spec:
+      containers:
+        - name: podsecurity-webhook
+          image: podsecurity-webhook:latest
+          ports:
+            - containerPort: 80
+          resources:
+            requests:
+              cpu: "0.5"
+              memory: "100Mi"
+            limits:
+              cpu: "1"
+              memory: "200Mi"
```

Comment:
These were totally arbitrary just so that _some_ values were set. If you or @liggitt have insight on more appropriate values, that would be cool. I also imagine these should be "kustomizable"

Reply:
The requests/limits seem high to me. It seems like it would be better to err low and let people bump these in big clusters, and perhaps not set limits by default (people can overlay limits if they want)